src/main/java/frc/robot/subsystems/conveyor/ConveyorConstants.java

```diff
@@ -9,5 +9,5 @@ public class ConveyorConstants {
   public static final double FRONT_GEAR_RATIO = Constants.PLACEHOLDER_DOUBLE,
       BACK_GEAR_RATIO = Constants.PLACEHOLDER_DOUBLE;
 
-  public static final double CONVEYOR_MOTOR_ROLLER_DIAMETER_IN = Constants.PLACEHOLDER_DOUBLE;
+  public static final double CONVEYOR_MOTOR_ROLLER_DIAMETER_IN = 1.1;
```

Comment:
This should be double checked according to the doc; either wait to merge or add a comment

Reply:
This one I won't think will matter a whole lot tbh. Like if the deviation is 1/8" or something it doesn't matter a whole lot

Reply:
Good point